### DISCUSSION TITLE
using in other pages with laravel 8

### DISCUSSION THREAD

**Discussion by aryala7** [spatie/ignition#25] [General]
hey guys I have a problem is using this package.
when I throw an exception in any page except index.php default Larave exception page will render. how should I fix this?
![image](https://user-images.githubusercontent.com/56627259/150805432-cdee7b9f-0fcb-493c-8432-3273741dc019.png)
will render the new page as excepted. but on other pages, my error page looks like this
![image](https://user-images.githubusercontent.com/56627259/150805595-33a3bacb-b5c0-481b-8198-ac512ba1fed3.png)

**Comment by freekmurze** [comment]
For Laravel, we recommend to use [spatie/laravel-ignition](https://github.com/spatie/laravel-ignition)

**Reply by aryala7** [reply]
thanks mate!
